src/containers/SearchPage/SearchContainer.tsx

```diff
@@ -206,54 +206,76 @@ const SearchContainer = ({
             )}
           </CompetenceWrapper>
         )}
-        <div aria-live="assertive">{loading && searchGroups.length === 0 && <Spinner />}</div>
-        {sortedFilterItems.length > 1 && (
-          <div>
-            <StyledText textStyle="title.small" id={resourceTypeFilterId}>
-              {t("searchPage.filterSearch")}
-            </StyledText>
-            <StyledCheckboxGroup
-              value={selectedFilters}
-              onValueChange={handleFilterToggle}
-              aria-labelledby={resourceTypeFilterId}
-            >
-              {sortedFilterItems.map((item) => (
-                <CheckboxRoot key={item.value} value={item.value} variant="chip">
-                  <CheckboxControl>
-                    <CheckboxIndicator asChild>
-                      <Done />
-                    </CheckboxIndicator>
-                  </CheckboxControl>
-                  <CheckboxLabel>{item.label}</CheckboxLabel>
-                  <CheckboxHiddenInput />
-                </CheckboxRoot>
+        <>
+          {loading ? (
+            <div aria-label={t("loading")} aria-busy={true}>
+              <Skeleton css={{ width: "20%" }}>
+                <StyledText textStyle="title.small">&nbsp;</StyledText>
+              </Skeleton>
+              <StyledCheckboxGroup>
```

Comment:
Dumt å bruke denne her? 🤔

Reply:
Kan hende at skjermlesere blir litt forvirret. Dette vil bli lest opp som en gruppe, noe det strengt tatt ikke er.

Reply:
Men når jeg tenker meg om så vil vel `aria-busy` sørge for at det ikke annonseres?

Reply:
Nei. aria-busy sier bare fra om at ting er busy :')

Reply:
hmmmm, sikker? men, mistenker at det ikke er riktig brukt sånn det er nå, men skal lese litt mer på det!
> When multiple parts of a live region need to be loaded before changes are announced to the user, set aria-busy="true" until loading is complete. Then set to aria-busy="false". This prevents assistive technologies from announcing changes before updates are done.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-busy